File: web/src/Models/Queries/StartupQuery/StartupQueryResponse.ts
import { AccountTypeDto } from '@/Models/Dtos/AccountTypeDto';
import { BankDto } from '@/Models/Dtos/BankDto';
import { CurrencyDto } from '@/Models/Dtos/CurrencyDto';

export interface StartupQueryResponse {
  accountTypes: AccountTypeDto[];
  banks: BankDto[];
  currencies: CurrencyDto[];
}

File: src/mark.davison.finance.models.dtos/Queries/StartupQuery/StartupQueryResponse.cs
namespace mark.davison.finance.models.dtos.Queries.StartupQuery;

public class StartupQueryResponse
{
    public List<BankDto> Banks { get; set; } = new();
    public List<AccountTypeDto> AccountTypes { get; set; } = new();
    public List<CurrencyDto> Currencies { get; set; } = new();
    public List<TransactionTypeDto> TransactionTypes { get; set; } = new();
}

